src/mpid/common/shm/mpidu_init_shm.c

```diff
@@ -79,10 +79,23 @@ int MPIDU_Init_shm_init(void)
     int mpi_errno = MPI_SUCCESS, mpl_err = 0;
     int local_leader;
     int rank;
+
```

Comment:
You may drop this commit [8fbbef Initialize interprocess lock in MPIDU_Init_shm_init]. 

I am refactoring that code and will take care of the atomic lock initialization (after this PR merges).

Reply:
I will drop this commit. Thanks!